### PR TITLE
disable splunkforwarder listener (OPS-3682)

### PR DESCRIPTION
**Pull request by nadeemshahzad**
Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR if it is code shared across multiple services and you don't own all of the services.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).

### CODE REVIEW

File: playbooks/roles/splunkforwarder/tasks/main.yml [hunk 1]
```diff
@@ -157,3 +157,14 @@
     - inputs
     - outputs
   notify: restart splunkforwarder
+
+- name: Disable splunkforwarder listener
+  blockinfile:
```
Comment (jdmulloy) [inline playbooks/roles/splunkforwarder/tasks/main.yml]: I'm not sure if we should be using blockinfile, but I think the only alternative is to manage the whole file. I'd like a second opinion from @edx/devops.

Comment (feanil) [reply playbooks/roles/splunkforwarder/tasks/main.yml]: +1 to managing the whole file.  Does this file already exist on the machines and have info in it?

Comment (jdmulloy) [reply playbooks/roles/splunkforwarder/tasks/main.yml]: I looked at a machine in stage yesterday. As far as I can tell it's not already managed by Ansible. It has content that I think is generated by the Splunk installer. A conf.d type setup to add an extra file would be nice, but I don't think Splunk has that.

Comment (nadeemshahzad) [reply playbooks/roles/splunkforwarder/tasks/main.yml]: We can update and deploy `server.conf` with our settings, Splunk automatically update and appends required settings in this file when it is restarted.
 https://answers.splunk.com/answers/223905/updating-serverconf-with-chef-how-can-i-generate-p.html

**Review by fredsmith — APPROVED** [review]

**Comment by nadeemshahzad** [comment]
@fredsmith @jdmulloy I have made few changes in PR as boot script is changed in new version of splunk forwarder. Can you please review these changes ? thanks

**Pull request closed without merging**